### PR TITLE
Fix newsletter

### PR DESCRIPTION
**Pull request by OswaldoPineda**
## Quick Info
Fixing weekly schedules, removing `whenever` gem due to incompatibility issues with Rails 7([some reports](https://github.com/javan/whenever/issues/848)) and configuring `rufus-scheduler` gem, also changing mail styles.

## How do you manually test this?
- Pull branch
- Add any user to the Newsletter model from the homepage or console
- run `rails c` from your terminal
- Execute `UserMailer.newsletter_mailer.deliver!`

## Screenshots (if apply)
image

### CODE REVIEW

File: app/assets/stylesheets/posts/_post.scss [hunk 1]
```diff
@@ -46,7 +46,8 @@
 .user-email {
   display: flex;
   width: fit-content;
-  color: $tomato !important;
+  color: black !important;
+  font-weight: 500;
```
Comment (JuanequeX) [inline app/assets/stylesheets/posts/_post.scss]: will be cool to sort alphabetically this block and use a color from `colors.scss`

File: app/assets/stylesheets/posts/_post.scss [hunk 2]
```diff
@@ -117,6 +118,10 @@
   display: flex;
   flex-direction: row;
   justify-content: space-between;
+
+  &__author > span {
+    color: #8f8f8f;
```
Comment (JuanequeX) [inline app/assets/stylesheets/posts/_post.scss]: same here, we can set this color in our `colors.scss` file

**Review by JuanequeX — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/posts/_post.scss (hunk 2 above)
Comment (OswaldoPineda) [reply app/assets/stylesheets/posts/_post.scss]: done!

File: app/assets/stylesheets/posts/_post.scss (hunk 1 above)
Comment (OswaldoPineda) [reply app/assets/stylesheets/posts/_post.scss]: thanks for the observation, now should be fixed!

**Review by JuanequeX — APPROVED** [review]

**Pull request merged**